tests/CMakeLists.txt

```
@@ -1 +1,50 @@
+# Add one test.
+#
+# The executable is always linked to 'kokkostools' and 'test_common'.
+#
+# Arguments:
+#   TARGET_NAME : name of the test
+#   SOURCE_FILE : source file, defaults to <TARGET_NAME>.cpp
+function(add_one_test)
+
+    cmake_parse_arguments(aot_args "" "TARGET_NAME;SOURCE_FILE" "" ${ARGN})
```

Maybe
```suggestion
    cmake_parse_arguments(TEST_ARGS "" "TARGET_NAME;SOURCE_FILE" "" ${ARGN})
```
to have consistent capitalization?

---

The convention I usually follow for the `PREFIX` given to `cmake_parse_arguments` is the following:
1. Take the first letter of each "work" in the function name (here `*a*dd_*o*ne_*t*est` leads to `aot`).
2. Append `_args`.

I think there are 2 advantages using this convention:
1. You reduce the probability that the chosen prefix matches a variable from the parent scope (note how easy it would be to get someone define `TEST_ARGS` somewhere in the parent scope for instance, while `aot_args` seems less probable IMO).
2. You get a consistent and indisputable recipe for the `PREFIX`.

What do you think?

---

IMO having the `PREFIX` in lower case also helps reading the function's code :eyeglasses:

---

FWIW I also dislike the prefix you picked